requirements/core.txt

```diff
@@ -6,6 +6,7 @@
 cartopy
 matplotlib<1.9
 netcdf4
+cftime
```

Comment:
@ocefpaf We're keeping there is alphabetical order, so could you move this to be after `cartopy` instead, thanks.

I'm looking at the fixes we need to cut `cf_units` 2.0, but once that is done, `cftime` will be pulled in by `cf_units`, but let's keep it as an explicit dependency here in order to keep things moving along